docs/instrumentation/python-fastapi-uvicorn.md

```diff
@@ -0,0 +1,93 @@
+---
+id: python-fastapi-uvicorn
+title: "Python FastAPI Uvicorn"
+slug: /instrumentation/python-fastapi-uvicorn
+---
+
+## Prerequisites
+
+Python 3
+
+## Installation
+
+1. Install dependencies:
+
+   ```shell
+   pip install opentelemetry-distro opentelemetry-exporter-otlp-proto-http opentelemetry-instrumentation-fastapi
+   opentelemetry-bootstrap -a install
+   ```
+
+2. Create a file tracing.js in your project directory, with the following content:
+
+   ```python title="tracing.js"
+    import os
+    from opentelemetry import trace
+    from opentelemetry.sdk.trace import TracerProvider
+    from opentelemetry.sdk.trace.export import (
+        BatchSpanProcessor,
+        ConsoleSpanExporter,
+        SimpleSpanProcessor,
+    )
+    from opentelemetry.exporter.otlp.proto.http.trace_exporter import OTLPSpanExporter
+
+
+    def init_tracing():
+        provider = TracerProvider()
```

Comment:
Also include changes from 6fd04ad. In fact, I missed adding these change in Python Sanic documentation. Please add there as well.